doc/user/truffleruby-additions.md

```diff
@@ -52,6 +52,44 @@ TruffleRuby provides these non-standard methods and classes that provide additio
 
 * `AtomicReference` is marshalable.
 
+### Concurrent Maps
+
+`TruffleRuby::ConcurrentMap` is a key-value data structure, like a `Hash` and using `#hash` and `#eql?` to compare keys and identity to compare values. Unlike `Hash` it is unordered. All methods on `TruffleRuby::ConcurrentMap` are thread-safe but should have higher concurrency than a fully syncronized implementation. It is intended to be used by gems such as Concurrent Map - please use via this gem rather than using directly in most cases.
```

Comment:
Clarify - gem name is `concurrent-ruby` and the class is `Concurrent::Map`

Reply:
Fixed.